src/cookbook/revoke-permissions.md

```diff
@@ -11,4 +11,32 @@ head:
 
 # How to Revoke Permissions
 
-TODO
\ No newline at end of file
+```rust
+fn revoke_permission_from_account(
+    iroha: &Client,
+) {
+    let revoke_permission_to_unregister_kingdom = Revoke::permission(
+        PermissionToken::new(
+            "CanUnregisterDomain".parse().unwrap(),
+            &json!({ "domain_id": "kingdom" }),
```

Comment:
Maybe, in case of `json!` in particular, it would make sense to specify the import path of this utility. For example, I was confused and thought it must be `serde_json::json!`, but it returns `serde_json::JsonValue`, while `PermissionToken::new` accepts `&str`.